ginga/misc/plugins/Cuts.py

```diff
@@ -120,6 +122,7 @@ def __init__(self, fv, fitsimage):
                                   show_cuts_legend=False, enable_slit=False)
         self.settings.load(onError='silent')
         self.colors = self.settings.get('colors', cut_colors)
+        self.show_legend = self.settings.get('show_cuts_legend', False)
```

Comment:
Just for future reference @pllim, if the preference is handled this way (by assigning it to an instance variable in the plugin object and thereafter only referring to the instance variable) then there is no way short of reloading the plugin to turn off the preference dynamically.  I think that this is fine for many preferences (where the user is likely to edit a file and restart the reference viewer), but it complicates logic for toggling the preference via a GUI.  In such cases it may be better to simply do:
```
if self.settings.get('show_cuts_legend', False):
```
to check the value.  This is also reinforces the idea that it is a preference that you are checking.